src/feeds/extractor/common.py

```diff
@@ -78,12 +78,33 @@ def _get_objects_from_word(cls, word):
     # 1行から word ごとに分割する
     @classmethod
     def _get_words_from_line(cls, line):
-        # " (" → " ( "
-        line = line.replace(' (', ' ( ')
-        # ") " → " ) "
-        line = line.replace(') ', ' ) ')
         return cls.word_extract_reg.findall(line)
 
+    @staticmethod
+    def _remove_parentheses(word):
+        if word[0] == '(' and word[-1] == ')':
+            return word[1:-1]
+        if word[0] == '[' and word[-1] == ']':
+            return word[1:-1]
+        if word[0] == '{' and word[-1] == '}':
+            return word[1:-1]
+        return word
+
+    @staticmethod
+    def _remove_parenthes(word):
+        if word[0] == '(' or word[0] == '[' or word[0] == '{':
+            word = word[1:]
+        if word[-1] == ')' or word[-1] == ']' or word[-1] == '}':
+            word = word[:-1]
+        return word
+
+    @staticmethod
+    def _refang_ioc(v):
+        v = v.replace('[', '').replace(']', '')
+        v = v.replace('(', '').replace(')', '')
+        v = v.replace('{', '').replace('}', '')
+        return v
```

Comment:
You can replace it with a regular expression or a for statement.

Reply:
Thanks @tkhsknsk .
I will try it. Please wait a couple of hours.

Reply:
@tkhsknsk 
Replace to some regular expressions. Please re-review it again.